packages/cli/src/commands/deploy/index.ts

```diff
@@ -0,0 +1,13 @@
+import { Command } from 'commander';
+
+import { generateSignature, register as _register } from '../../register-command';
+
+import * as spec from './spec';
+
+export { name, description } from './spec';
+
+export const signature = generateSignature(spec.name, spec.args);
+
+export function register(program: Command) {
+  _register(program, spec, () => import('./action'));
```

Comment:
Register is always called, but its action is only called when the command is invoked on the CLI. I used lazy imports so that command-specific code (prompts and actions) is only loaded if needed. I expect this to improve the loading times of the CLI, once everything is loaded lazily. I haven't measured it though and we can discuss whether it's worth the effort.

https://github.com/OpenZeppelin/openzeppelin-sdk/blob/a851d4ce1cc475e1b227c0decfe891b90b90a35a/packages/cli/src/register-command.ts#L53-L54

Reply:
Makes sense, thanks!